client/client.go

```diff
@@ -251,6 +251,22 @@ func (c *RemoteServer) Decrypt(req core.DecryptRequest) (*core.ResponseData, err
 
 }
 
+// Decrypt issues an decrypt request to the remote server
```

Comment:
Thanks - fixed in next push. I'm wondering if there's a better name for this endpoint - decrypt-sign could be confusing - but I couldn't come up with anything.